### PR TITLE
Restructuring folders

### LINKED ISSUE #67: restructuring the folders

### PR DESCRIPTION
**Pull request by chiturca**
# Description

- Folders reorganized as img below (pages' main container components renamed, index.jsx name has been thought but its confusing for us I think)
- config file tried to create but tailwind config gives error, so it left out
- svg folders reorganized
- Location and Partners has been added to AboutPage
- Location is responsive now
- Title renamed as "The Incredibles RefuBook"

## Related Issue

- Fixes #67 

## Screenshots

folders
contact-home
location-title

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested locally
- [ ] Added tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [x] I've run linter, prettier, and removed all `console.log`
- [x] I've reverted all the files back that are not related to the issue (App.js, index.js, ...)

### CODE REVIEW

**Comment by chiturca** [comment]
- Opps, lets solve #70 first, then I will update here

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React Redux App</title>
+    <title>The Incredibles RefuBook</title>
```
Comment (ahmadmhd357) [inline public/index.html:27]: Finally 👍

**Review by ahmadmhd357 — APPROVED** [review]
nice work thank you so much, we really needed

**Comment by ahmadmhd357** [comment]
I think we still have some conflicts here so whoever going to merge this pull request please make sure to resolve first

**Comment by chiturca** [comment]
yeah I solved them :)

**Comment by ahmadmhd357** [comment]
> yeah I solved them :)

yes thank you 👍

**Review by batoulst — APPROVED** [review]
Nice work 👏  finally organized 
thank you

**Pull request merged**